bnelson777/id.ly
# Login/Register components working on dev options

Modified version of original IL-130

Login/Register components updated and password functionality working.

Testing
1. Change to dev mode
2. Set two different passwords for reg screen and ensure you get a prompt to re-enter.
3. Set the same passwords and make sure you get successful alert.
4. Go to login screen (also in dev mode) and enter incorrect password. Make sure error message pops up.
5. Enter correct password and make sure you are navigated back to home screen.

![screen shot 2018-06-03 at 7 38 37 pm](https://user-images.githubusercontent.com/32649350/40895713-cb8cdcb4-6765-11e8-925c-168c9620cd9d.png)
![screen shot 2018-06-03 at 7 38 48 pm](https://user-images.githubusercontent.com/32649350/40895714-cba954f2-6765-11e8-82e1-e9522a69886f.png)
